### PR TITLE
[THEMES] Octovita Theme

### PR DESCRIPTION
**Pull request by NikkiNext**
## Purpose

I made a Octovita theme. I was inspired by Windows 8 and Gnome 3. Octovita has a dark and light version. The name comes from "Octo" - Windows 8, "Vita"- Life

## Proposed changes

 - Add Octovita Theme resources

## TODO

I made this PR in order for you to understand the patch

P. S. The star in the Start button I temporarily made. Because I couldn't make the ReactOS logo in the Fluent style. I didn't add the wallpaper from the picture below to the repository.

I wish you a Happy New Year. 

preview1
preview2

### CODE REVIEW

**Comment by NikkiNext** [comment]
I transferred dollars to you through my friend for the development of ReactOS

donate

**Comment by HBelusca** [comment]
I wonder whether there could be a dark version of this theme :)

**Comment by NikkiNext** [comment]
> I wonder whether there could be a dark version of this theme :)

The dark version of the theme will be

**Comment by NikkiNext** [comment]
@learn-more @binarymaster I have set up an email and a name for PR

File: sdk/cmake/CMakeMacros.cmake [hunk 1]
```diff
@@ -255,7 +255,8 @@ macro(dir_to_num dir var)
         set(${var} 80)
     elseif(${dir} STREQUAL reactos/winsxs/arm64_microsoft.windows.gdiplus_6595b64144ccf1df_1.0.14393.0_none_deadbeef)
         set(${var} 81)
-
+	elseif(${dir} STREQUAL reactos/Resources/Themes/Octovita)
```
Comment (Saibamen) [inline sdk/cmake/CMakeMacros.cmake]:
```suggestion
    elseif(${dir} STREQUAL reactos/Resources/Themes/Octovita)
```

File: sdk/cmake/CMakeMacros.cmake [hunk 2]
```diff
@@ -255,7 +255,8 @@ macro(dir_to_num dir var)
...
+        set(${var} 82)
```
Comment (learn-more) [inline sdk/cmake/CMakeMacros.cmake]: This also needs an edit in `boot/bootdata/packages/reactos.dff.in`

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Thanks for your contribution, but
we agreed in the past that we do not want to add any more Themes in the ros source tree.
At least we won't add any without deleting another one at the same time.
So unless there is common sense between the ros developers about what to drop, you have to commit this into another repository.

**Comment by NikkiNext** [comment]
@learn-more I added the design theme to the file "reactos.dff.in"

**Comment by Swyter** [comment]
This one seems really well made, especially if it has light and dark variants. At the very least it can probably be added to RAPPS for a one-click install.

**Comment by GatoGit21** [comment]
Honestly Octovita looks beautiful! Good job.
I suggest you that if you still want to publish this theme, create an installer compatible with ROS with your theme's files, upload it to the Internet, create a new txt file in RAPPS, write the download link to your installer and the SHA Key to the file, and commit it to rapps-db. If you created correctly the txt file and complied with the mods' requests, they will merge your file to the db.
This way Octovita could be downloaded and used by ROS users.

**Comment by Swyter** [comment]
@NikkiNext Yeah, even a self-extracting archive pointed to the right default folder would do the trick. You can contact the ReactOS devs easily through IRC or Matrix, and they can give you some tips. 

The RAPPS file can be as easy to do as this one: https://github.com/reactos/rapps-db/blob/8b572a5644904327f8ff812f1f80351452265af8/longhorn5048.txt

Once it can be installed by normal ReactOS users and they like what they see it will be much easier to convince them to add it to normal builds. Maybe replacing some other theme.

So yeah, I'd do some «lobbying» over Matrix chat. Get it into the project in small increments. 

Your theme is pretty good, I think this was just closed because they already have a lot of themes in the ISO and they wanted to keep them lean. Good luck. :)

**Pull request closed without merging**